how-not-to-be-a-dinosaur.md

```diff
@@ -0,0 +1,135 @@
+
+So I did two things: I decided to start learning all these new tools and frameworks and more importantly I started to ask for help. But the most important thing I did: I told my colleagues how I was feeling and how insecure I was about everything. They were very supportive and this gave me the confidence to go for it.
+
+One of the most important things they gave me is that they showed me that I did make progress. A lot, actually. And that it's not the end of the world not knowing everything. As long as I kept learning, things will work out in the end, was the message I got.
+
+Also, I realised I did not have to start all over again. It might have felt like it, but I still had a big headstart with my years of experience in programming overall. It was just a new framework and new tools that I needed to learn, not learning how to speak and not the first tool I ever used.
+
+It will take some time to learn all these new things. And that's okay. Good things take time. I should have realised this earlier since it's litteraly written on my body, but sadly it's not that obvious sometimes.
+
+![](https://raw.githubusercontent.com/MyOnlineStore/blogs/Dinosaurusblog/public/johan_arm.png)
+
+_Yes, this is also me._
+
+## Okay great, challenge accepted! Where to begin?
+
+First I made a list of all the tools we use. There were quite a few things for me to learn: Typescript, Babel, Webpack, React, Apollo, Redux, GraphQL, Yarn and many more. This list was somewhat overwhelming until I started to put them into two groups: things I needed to learn right from the start and things I could figure out later. It turned out the first list is not that big and and a lot more do-able.
+
+For example: I now know how to set up a project using Babel, although I have no idea about the inner workings. And that's okay, I will learn it some day. Until then it's perfectly fine to ask a colleague who does get it.
+
+Bottomline is that it is okay to not know things. It's not the amount of things you know that make you a senior developer, it's the way you handle things [that make you a senior](https://medium.com/better-programming/how-to-be-a-good-senior-developer-958948e02ada). In my opinion.
+
+I have always tried to take some time for personal growth once in a while, but this was only a few spare hours now and then. However, since a few months our company [MyOnlineStore](https://www.myonlinestore.com/) is doing so called [Ship-it days](https://www.atlassian.com/company/shipit) where we developers can do whatever we want as long as it's somehow work related. This is a great way for me to learn stuff I might not naturally learn during everyday work. When I work in our normal codebase I just run `yarn dev` and storybook starts and that's it. But on these ship-it days I create new repo's, install packages and set them up. That is a great way to learn new things.
```

Comment:
```suggestion
I have always tried to take some time for personal growth once in a while, but this was only a few spare hours  every now and then. However, since a few months our company [MyOnlineStore](https://www.myonlinestore.com/) is doing so called [Ship-it days](https://www.atlassian.com/company/shipit) where we developers can do whatever we want as long as it's somehow work related. This is a great way for me to learn stuff I might not naturally learn during everyday work. When I work in our normal codebase I just run `yarn dev`, storybook starts and that's it. But on these ship-it days I create new repo's, install packages and configure everything myself. That is a great way to learn new things.
```